auter

```diff
@@ -158,16 +160,22 @@ function run_script {
       if [[ -x "$script" ]] && [[ -f "$script" ]]; then
         logit "INFO: Running $phase script $script"
         $script
-        local RC=$?
-        if [[ "$RC" -ne 0 ]]; then
-          logit "ERROR: $phase script $script exited with non-zero exit code $RC. Aborting auter run."
-          quit 8
+        rc=$?
+
+        if [[ "$rc" -ne 0 ]]; then
+          logit "ERROR: $phase script $script exited with non-zero exit code $rc. Aborting auter run."
```

Comment:
If --ignore-script-failure is set then the phrase 'Aborting auter run' will not be true

Reply:
Good catch.